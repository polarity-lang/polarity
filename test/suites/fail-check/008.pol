data A {a}
data B {b} // annotate the type of b explicitly
data C (a : A) {}

let f : C (b) {b}
