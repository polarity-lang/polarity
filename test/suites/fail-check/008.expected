T-002

  × The following terms are not equal:
  │   1: B
  │   2: A
  │ 
   ╭─[008.pol:2:1]
 2 │ data B {b} -- annotate the type of b explicitly
 3 │ data C (a : A) {}
   ·             ┬
   ·             ╰── Source of (2)
 4 │ 
 5 │ let f : C (b) {b}
   ·            ┬
   ·            ╰── While elaborating
   ╰────
